apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -1368,19 +1368,20 @@ to_existing_atom(X) ->
         {error, _} -> throw(bad_atom)
     end.
 
-validate_referenced_connectors(Type0, ConnectorName0, BridgeName) ->
+validate_referenced_connectors(BridgeType, ConnectorNameBin, BridgeName) ->
     %% N.B.: assumes that, for all bridgeV2 types, the name of the bridge type is
     %% identical to its matching connector type name.
     try
-        Type = to_existing_atom(Type0),
-        ConnectorName = to_existing_atom(ConnectorName0),
-        case emqx_config:get([connectors, Type, ConnectorName], undefined) of
+        ConnectorType = bridge_v2_type_to_connector_type(to_existing_atom(BridgeType)),
+        ConnectorName = to_existing_atom(ConnectorNameBin),
+        case emqx_config:get([connectors, ConnectorType, ConnectorName], undefined) of
             undefined ->
                 {error, #{
                     reason => "connector_not_found_or_wrong_type",
-                    type => Type,
+                    connector_name => ConnectorName,
+                    connectortype => ConnectorType,
```

Comment:
Nit:

```suggestion
                    connector_type => ConnectorType,
```

Reply:
But, on second thought, since we adopted the convention that the connector type name must always match the bridge type name, maybe we could keep this key as `type`?  This would also make this error map match the one below from the `bad_atom` catch clause.

Reply:
Please have another look now. I’ve simplified/refactored this piece of code because I don’t think there should be two places where we generated more or less the same error. Now there can’t be `connector_type` anymore since this conversion is the one that might have failed. But I tried to align the parameters to the log message with the parameter names of the function call for better readability.